src/sentry/migrations/0359_duplicate_perf_issue_events_issue_platform.py

```diff
@@ -0,0 +1,222 @@
+from sentry.issues.ingest import process_occurrence_data
+from sentry.issues.issue_occurrence import IssueOccurrence, IssueOccurrenceData
+from sentry.models import Group
+from sentry.new_migrations.migrations import CheckedMigration
+from sentry.snuba.dataset import Dataset, EntityKey
+
+
+def backfill_eventstream(apps, schema_editor):
+    """
+    Backfills Performance-issue events from the transaction table to the IssuePlatform dataset(search_issues).
+    1. Read all transactions that back a performance-issue(s) with the following criteria:
+        a. non-null group_ids column
+        b.
+    2. Prepare and format each transaction to be submitted through the eventstream.
+    3. Send it through the eventstream.
+    """
+
+    # TODO: need to make sure event_type is correct in eventstream.send()
+    from sentry.issues.ingest import send_issue_occurrence_to_eventstream
+
+    # retrieve rows
+    rows = _query_performance_issue_events()
+
+    for row in rows:
+        try:
+            # don't need to store in node_store since it should be there already, maybe check and write if doesn't exists
+            # create issue occurrence
+            # save issue occurrence
+            event, occurrence_data, group_id = _map_performance_issue_row_to_params(row)
+
+            group: Group = Group.objects.get(id=group_id)
+
+            occurrence, group_info = _save_issue_occurrence(occurrence_data, event, group)
+
+            send_issue_occurrence_to_eventstream(event, occurrence, group_info)
+        except Exception:
+            print("Failed to process row")  # noqa: S002
+
+
+def _map_performance_issue_row_to_params(
+    row: Mapping[str, Any]
+) -> Tuple[Event, IssueOccurrenceData, int]:
+    # we need to do the inverse of TransactionProcessor (Event -> EventStreamPayload -> Row):
+    # Row -> Event
+    # some columns in the rows have been extracted out from the Event interfaces to a top-level row and stripped out
+    # we might need to put them back in
+    data = {
+        "platform": "",
+        "message": "",
+        "tags": [],
+        "breadcrumbs": {},
+        "contexts": {},
+        "extra": {},
+        "logger": "",
+        "metadata": {},
+        "request": {},
+        "spans": [],
+        "transaction": "",
+        "type": "",
+        "user": {},
+        "measurements": {},
+        "breakdowns": {},
+    }
+    project_id = row["project_id"]
+    event_id = row["event_id"]
+
+    event: Event = Event(project_id=project_id, event_id=event_id, data=data)
+    occurrence_data: IssueOccurrenceData = IssueOccurrenceData(
+        id="",  # TODO: need to generate this
+        project_id=project_id,
+        event_id=event_id,
+        fingerprint=_retrieve_fingerprints(event_id, project_id),
+        issue_title="",
+        subtitle="",
+        resource_id=None,
+        evidence_data="",
+        evidence_display="",
+        type="",
+        detection_time="",
+        level="",
```

Comment:
We can get some of this info from the `evidence` I think. Maybe grab some from the group as well, since culprit etc are set in the group data.